pkg/controllers/membercluster/v1alpha1/membercluster_controller.go

```diff
@@ -567,7 +567,7 @@ func markMemberClusterUnknown(recorder record.EventRecorder, mc apis.Conditioned
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
-	r.recorder = mgr.GetEventRecorderFor("memberCluster")
+	r.recorder = mgr.GetEventRecorderFor("mcv1alpha1")
```

Comment:
Just a nit: maybe it's better to use the full name here.

Reply:
Same for the v1beta1 controller.

Reply:
Will handle in the PR for v1beta1 IMC controller